### DISCUSSION TITLE
where is my old project?

### DISCUSSION THREAD

**Discussion by Minnefekkes** [asreview/asreview#1303] [Q&A]
I was working on a review with ASReview in 2021.
I have a new laptop and installed ASreview again, but now i dont know how to find my project again.

**Comment by Rensvandeschoot** [comment]
Did you [export](https://asreview.readthedocs.io/en/latest/manage.html#export-project) your old project and store it somewhere?

**Comment by Minnefekkes** [comment]
Ik heb alleen een excel export file met de resultaten.
Maar dacht dat ik gewoon weer bij mijn project kan inloggen om er een update van abtsracts aan toe te voegen. Maar geen ideel hoe ik mijn oude project weer kan openen in ASReviw.  Ik neem aan dat dat nog ergens op een server staat?

Groet
Minne




Prof. Dr. M. (Minne) Fekkes
Bijzonder hoogleraar Sociale Vaardigheden en Weerbaarheid, Universiteit van Amsterdam

Senior wetenschappelijk onderzoeker
TNO Child Health, Leiden



T +31 (0)6 340 565 95
E ***@***.******@***.***>






***@***.***<http://www.tno.nl/>

Dit bericht kan informatie bevatten die niet voor u is bestemd. Indien u niet de geadresseerde bent of dit bericht abusievelijk aan u is toegezonden, wordt u verzocht dat aan de afzender te melden en het bericht te verwijderen. TNO aanvaardt geen aansprakelijkheid voor de inhoud van deze e-mail, de wijze waarop u deze gebruikt en voor schade, van welke aard ook, die verband houdt met risico's verbonden aan het elektronisch verzenden van berichten.

**Comment by Rensvandeschoot** [comment]
If you have used ASReview on your local laptop, the project folder is stored locally and you should open your old laptop to obtain the project folder. If you have used a server installation, please contact your IT department because they might have access to your old folders. 

However, If you just want to continue screening, you can simply import the excel file, and ASReview will recognize all the labels you provided in 2021. If you want to merge files, you might be interested in the merge option of [Datatools](https://github.com/asreview/asreview-datatools#asreview-datatools).
